audio/out/ao_jack.c

```diff
@@ -212,11 +242,10 @@ static int init(struct ao *ao)
 
     ao->samplerate = jack_get_sample_rate(p->client);
 
-    jack_latency_range_t jack_latency_range;
-    jack_port_get_latency_range(p->ports[0], JackPlaybackLatency,
-                                &jack_latency_range);
-    p->jack_latency = (float)(jack_latency_range.max + jack_get_buffer_size(p->client))
-                      / (float)ao->samplerate;
+    p->buffer_size       = ATOMIC_VAR_INIT(0);
+    p->graph_latency_max = ATOMIC_VAR_INIT(0);
```

Comment:
Probably not needed (not sure, would have to check the standard). On the other hand, using the macro like this might not be allowed, because it's supposed to be an initializer only (again would have to check the standard).